features/apimgt/org.wso2.carbon.apimgt.admin.feature/src/main/resources/admin_new/source/src/app/components/Throttling/Subscription/List.jsx

```diff
@@ -0,0 +1,297 @@
+                defaultMessage: 'Time Unit',
+            }),
+            options: {
+                filter: true,
+                sort: false,
+            },
+        },
+    ];
+
+    const emptyBoxProps = {
+        content: (
+            <Typography variant='body2' color='textSecondary' component='p'>
+                <FormattedMessage
+                    id='AdminPages.Microgateways.List.empty.content.microgateways'
+                    defaultMessage={'It is possible to create a Microgateway distribution '
+                    + 'for a group of APIs. In order to group APIs, a label needs to be created'
+                    + ' and attached to the APIs that need to be in a single group.'}
+                />
+            </Typography>),
+        title: (
+            <Typography gutterBottom variant='h5' component='h2'>
+                <FormattedMessage
+                    id='AdminPages.Microgateways.List.empty.title.microgateways'
+                    defaultMessage='Microgateways'
+                />
+
+            </Typography>),
+    };
+
+    const addButtonOverride = (
+        <RouterLink to='/throttling/subscription/add'>
+            <Button variant='contained' color='primary'>
+                <FormattedMessage
+                    id='Throttling.Subscription.Policy.List.addButtonProps.title'
+                    defaultMessage='Add Policy'
+                />
+            </Button>
+        </RouterLink>
+    );
+
+    /**
+     * Populate subscription policies
+     * @returns {Promise} The list of subscription policies
+     */
+    function apiCall() {
+        let subscriptionThrottlingvalues;
+        return restApi.getSubscritionPolicyList().then((result) => {
+            setSubscriptionThrottlingPolicyList(result.body.list);
+            const subscriptionPolicies = result.body.list.map((obj) => {
+                if (obj.defaultLimit.requestCount !== null) {
+                    return {
+                        policyName: obj.policyName,
+                        quotaPolicy: obj.defaultLimit.requestCount.type,
+                        quota: obj.defaultLimit.requestCount.requestCount,
+                        unitTime: obj.defaultLimit.requestCount.unitTime + ' '
+                        + obj.defaultLimit.requestCount.timeUnit,
+                        rateLimit: obj.rateLimitCount,
+                        timeUnit: obj.rateLimitTimeUnit,
+                    };
+                } else {
+                    return {
+                        policyName: obj.policyName,
+                        quotaPolicy: obj.defaultLimit.bandwidth.type,
+                        quota: obj.defaultLimit.bandwidth.requestCount,
+                        unitTime: obj.defaultLimit.bandwidth.unitTime + ' '
+                        + obj.defaultLimit.requestCount.timeUnit,
+                        rateLimit: obj.rateLimitCount,
+                        timeUnit: obj.rateLimitTimeUnit,
```

Comment:
shall we send the policyId also here? Then we can access it through dataRow in delete component